src/Sentry/Internal/Extensions/HashExtensions.cs

```diff
@@ -2,11 +2,11 @@ namespace Sentry.Internal.Extensions;
 
 internal static class HashExtensions
 {
-    public static string GetHashString(this string str)
+    public static string GetHashString(this string str, bool upperCase = true)
```

Comment:
Why is it `true` by default?

Reply:
Because that was the previous behavior before refactoring.